modules/5_install/templates/install_inventory

```diff
@@ -1,5 +1,8 @@
 [bastion]
 ${bastion_ip} ansible_connection=local
+%{ for ip in bastion_ip ~}
+${ip} ansible_connection=ssh ansible_user=root
```

Comment:
Should the `ansible_user` come from the `rhel_username` variable used in var.tfvars ?

Reply:
Yes.. will create an issue for this to be fixed across repos.

Reply:
https://github.com/ocp-power-automation/ocp4-upi-powervs/issues/116

Reply:
Will take it in new PR. Okay?

Reply:
yeah that should be fine.